src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="moyo-container">
+      <header class="main-header">
+        <img src="images/logo.png" alt="moyo-logo1"
+         class="moyo-logo1">
+        <nav class="main-nav">
+          <ul class="main-nav-list">
+            <li>
+              <a href="#" class="is-active">Apple</a>
+            </li>
+            <li>
+              <a href="#">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li>
              <a href="#" class="is-active">Apple</a>
            </li>

            <li>
              <a href="#">Samsung</a>
            </li>
```
add empty line between sibling tags to increase readability